.env

```diff
@@ -0,0 +1 @@
+FLINK_VERSION=1.17
```

Comment:
I don't agree with this so much, since the version is also dependent to the version in `build.gradle`.

I would rather use the same trick with a gradle task, as we did with the project version` to print the version from there in the scripts and use it.

Reply:
Excellent, thanks. See b29d18e2ba4.